modules/loader.js

```diff
@@ -660,9 +660,8 @@ __private.loadBlocksFromNetwork = function(cb) {
 		async.whilst(
 			() => !loaded && errorCount < 5,
 			next => {
-				const peer =
-					network.peers[Math.floor(Math.random() * network.peers.length)];
 				let lastBlock = modules.blocks.lastBlock.get();
+				const peer = getGoodPeer(lastBlock.height, network.peers);
```

Comment:
Why are we not using the same `findGoodPeers` function that we already have?

Reply:
Reusing `findGoodPeers` adds the calculation overhead to that issue. For keeping the responsibility for selecting good peers I agree it makes sense. 

I will reuse it then.